File: day56/Homework/N8.py
def remove_smallest(arr):
    if not arr:
        return []
    min_val = min(arr)
    return [x for x in arr if x != min_val]

print(remove_smallest([1, 2, 3, 4, 5]))
print(remove_smallest([5, 3, 2, 1, 4]))
print(remove_smallest([2, 2, 1, 2, 1]))

File: day56/Classwork/N8.js
function removeSmallest(arr) {
    if (arr.length === 0) return [];

    const minIndex = arr.indexOf(Math.min(...arr));
    return arr.slice(0, minIndex).concat(arr.slice(minIndex + 1));
}
console.log(removeSmallest([1, 2, 3, 4, 5]));
console.log(removeSmallest([5, 3, 2, 1, 4]));
console.log(removeSmallest([2, 2, 1, 2, 1]));
